website/static/js/comment.js

```diff
@@ -62,6 +115,58 @@ var exclusifyGroup = function() {
     }
 };
 
+ko.subscribable.fn.convertHtmlToMarkdown = function(commentContent) {
+    return ko.computed(function() {
```

Comment:
Why does this function signature have to be so complicated? Why can't we just have ...

```javascript
function convertHtmlToMarkdown(html) {
    ...
    return markdown;
}
```

? Then in the tests we can have the more thorough tests (making sure that `@` works, `+` works, etc.) only once—right now in https://github.com/CenterForOpenScience/osf.io/commit/913db15a6da64379504226baf483fb019aef673c we test the same things twice—with just one or two tests to make sure that the conversion code is getting called _at all_ by the higher-order knockout wrappers. Does that make any sense?